draft-ietf-jose-hpke-encrypt.md

```diff
@@ -476,6 +489,20 @@ When using Key Encryption, the strength of the content encryption algorithm shou
 
 This document adds entries to {{JOSE-IANA}}.
 
+## Updates to "JSON Web Key Use" Registry
+
+The "JSON Web Key Use" registry is updated as follows:
+
+   o  Use Member Value: "HPKE-Sender"
+   o  Use Description: Key for HPKE sender role (encapsulation)
```

Comment:
Maybe "Key for HPKE sender role (authentication)"

Reply:
Fixed.